Docs/Interaction_Logs.md

```diff
@@ -0,0 +1,42 @@
+## Interaction Log service
+### Background -
+The interaction log service will notify all the Firebolt calls getting invoked in the platform whether the Firebolt call is made manually by any action or internally made by any application or service. Whenever the platform identifies a firebolt call, the interaction log service will notify that.
+### Uses -
+The Interaction log service is essential for tracking method calls. When a method is called, it internally triggers/invokes other Firebolt methods. This service can be used to obtain the list of internal method calls.
```

Comment:
Interaction log service is used to capture the various firebolt calls invoked during a test execution. This service will notify all the Firebolt calls getting invoked in the platform if the Firebolt call is called manually by any action or internally called by any application or service.

Reply:
Uses 
The Interaction log service is essential for tracking all the firebolt calls made by the platform. When a method is called, it may internally trigger/invoke other Firebolt methods. This service can be used to obtain the list of internal method calls and validate them if needed.

Reply:
done